### PR TITLE
Python3 compatibility

### PR DESCRIPTION
**Pull request by boojums**
Small fixes for python3 compatibility.

### CODE REVIEW

File: pyrate/utils/__init__.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ def build_oauth1(client_key, client_secret, resource_owner_key, resource_owner_s
 
 def clean_dict(dirty_dict):
     """Cleans a dictionary from keys with empty string values"""
-    return dict((k, v) for k, v in dirty_dict.iteritems() if v)
+    return dict((k, v) for k, v in dirty_dict.items() if v)
```
Comment (Chive) [inline pyrate/utils/__init__.py]: this unfortunately will copy the entire dict into memory in Py2 - could you please add six to the app and use [`six.iteritems`](Hey there, thanks for your PR!) instead? Thanks!

**Review by Chive — CHANGES_REQUESTED** [review]
Hey there, thanks for your PR. Please have a look at the comment I made, thanks!

**Comment by boojums** [comment]
Ok, I think that should do it. I'm not very familiar with py2/3 issues so maybe there's more to it?

**Review by Chive — APPROVED** [review]
Awesome, thank you @boojums

**Pull request merged**